jaxrs-tck/src/main/java/jakarta/ws/rs/tck/jaxrs21/ee/client/executor/rx/JAXRSClientIT.java

```diff
@@ -0,0 +1,772 @@
+  }
+
+  /*
+   * @testName: postWithResponseClassTest
+   * 
+   * @assertion_ids: JAXRS:JAVADOC:1131;
+   * 
+   * @test_Strategy: Invoke HTTP post method for the current request
+   */
+  @Test
+  public void postWithResponseClassTest() throws Fault {
+    super.postWithResponseClassTest();
+  }
+
+  /*
+   * @testName: postWithGenericTypeStringTest
+   * 
+   * @assertion_ids: JAXRS:JAVADOC:1131;
+   * 
+   * @test_Strategy: Invoke HTTP post method for the current request
+   */
+  @Test
+  public void postWithGenericTypeStringTest() throws Fault {
+    super.postWithGenericTypeStringTest();
+  }
+
+  /*
+   * @testName: postWithGenericTypeResponseTest
+   * 
+   * @assertion_ids: JAXRS:JAVADOC:1131;
+   * 
+   * @test_Strategy: Invoke HTTP post method for the current request
+   */
+  @Test
+  public void postWithGenericTypeResponseTest() throws Fault {
+    super.postWithGenericTypeResponseTest();
+  }
+
+  // ------------------------------------------------------------------
+  // ---------------------------PUT -----------------------------------
+  // ------------------------------------------------------------------
+
+  /*
+   * @testName: putTest
+   * 
+   * @assertion_ids: JAXRS:JAVADOC:1131;
+   * 
+   * @test_Strategy: Invoke HTTP PUT method for the current request
+   */
+  @Test
+  public void putTest() throws Fault {
+    super.putTest();
+  }
+
+  /*
+   * @testName: putWithStringClassTest
+   * 
+   * @assertion_ids: JAXRS:JAVADOC:1131;
+   * 
+   * @test_Strategy: Invoke HTTP put method for the current request
+   */
+  @Test
+  public void putWithStringClassTest() throws Fault {
+    super.putWithStringClassTest();
+  }
+
+  /*
+   * @testName: putWithResponseClassTest
+   * 
+   * @assertion_ids: JAXRS:JAVADOC:1131;
+   * 
+   * @test_Strategy: Invoke HTTP put method for the current request
+   */
+  @Test
+  public void putWithResponseClassTest() throws Fault {
+    super.putWithResponseClassTest();
+  }
+
+  /*
+   * @testName: putWithGenericTypeStringTest
+   * 
+   * @assertion_ids: JAXRS:JAVADOC:1131;
+   * 
+   * @test_Strategy: Invoke HTTP put method for the current request
+   */
+  @Test
+  public void putWithGenericTypeStringTest() throws Fault {
+    super.putWithGenericTypeStringTest();
+  }
+
+  /*
+   * @testName: putWithGenericTypeResponseTest
+   * 
+   * @assertion_ids: JAXRS:JAVADOC:1131;
+   * 
+   * @test_Strategy: Invoke HTTP put method for the current request
+   */
+  @Test
+  public void putWithGenericTypeResponseTest() throws Fault {
+    super.putWithGenericTypeResponseTest();
+  }
+
+  // ------------------------------------------------------------------
+  // ---------------------------TRACE -----------------------------------
+  // ------------------------------------------------------------------
+
+  /*
+   * @testName: traceTest
+   * 
+   * @assertion_ids: JAXRS:JAVADOC:1131;
+   * 
+   * @test_Strategy: Invoke HTTP trace method for the current request
+   */
+  //@Test
```

Comment:
Why are these tests commented out? They are not in ts.jtx

Reply:
Yes. I missed it in this PR. I have another PR ready https://github.com/eclipse-ee4j/jaxrs-api/pull/1045/ to implement the excluded list from ts.jtx. Please ignore them in this PR.

Reply:
I have implemented the exclude list in a separate PR, will be ready once this PR is merged. The commit is https://github.com/eclipse-ee4j/jaxrs-api/pull/1045/commits/2d71392d29455c4a146143c4b50a2d7ca685f26d

Reply:
Test fixed by latest commit.